Library/Formula/fb-infer.rb

```diff
@@ -0,0 +1,53 @@
+class FbInfer < Formula
+  desc "Static analyzer for Java, C and Objective-C"
+  homepage "http://fbinfer.com/"
+  url "https://github.com/facebook/infer/releases/download/v0.5.0/infer-osx-v0.5.0.tar.xz"
+  sha256 "6a8547ac0b75a5e2bbeccae2169e39f753a60adbcacb6c94599fd31343a71ce7"
+  head "https://github.com/facebook/infer.git"
+
+  option "with-clang-only", "Installs Infer with only Clang support."
+  option "with-java-only", "Installs Infer with only Java support."
```

Comment:
Please omit the full stop at the end of the option descriptions. We also tend to phrase these things `Build with <feature>` (no need to repeat the formula name here), so I'd suggest:

- `Build with C/Objective-C analyzer only` (this makes it also clearer than just stating `Clang`)
- `Build with Java analyzer only`